package.json

```diff
@@ -46,7 +46,9 @@
   },
   "dependencies": {
     "es6-promisify": "^6.1.0",
-    "minimatch": "^3.0.4"
+    "minimatch": "^3.0.4",
+    "path": "^0.12.7",
```

Comment:
Why are we adding `path` vs using node's or the bundler's equivalent?

Reply:
My bad, removed it now